### PR TITLE
Patient counter

### PR DESCRIPTION
**Pull request by jeraldkiew**
- add total number of patients to GUI (for 'list' and 'find')
-> scrapped global variable as hard to track deletes and buggy-er compared to get size way

- partial fix of tests 
error: Can't add different class with same name: seedu/address/commons/core/Messages
 no idea burnt my time lol

- small fixes
-> fix error msg for name
-> rename jar file to medinfo.jar

- UG
-> done other than photos and discharge

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ MedInfo is a **desktop app for managing patients, optimized for use via a Comman
 
    - `list` : Lists all patients.
 
-   - `add nric/S1234567A name/John Doe cond/Ligma` : Adds a patient named `John Doe` to MedInfo.
+   - `add nric/S1234567A` : Adds a patient named `John Doe` to MedInfo.
```
Comment (nramapurath) [inline docs/UserGuide.md:30]: Isn't name/ a required argument in our command format? You might've accidentally missed typing it here :)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -44,17 +44,17 @@ MedInfo is a **desktop app for managing patients, optimized for use via a Comman
 **:information_source: Notes about the command format:**<br>
 
 - Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  e.g. in `add name/NAME`, `NAME` is a parameter which can be used as `add name/John Doe`.
```
Comment (nramapurath) [inline docs/UserGuide.md:47]: I think for consistency's sake maybe we should write this example as a valid command too (e.g. `add name/NAME nric/NRIC`)..

File: docs/UserGuide.md [hunk 3]
```diff
@@ -77,16 +77,16 @@ Format: `help`
 
 Adds the patient (NRIC, name and condition).
```
Comment (nramapurath) [inline docs/UserGuide.md:78]: Should we change condition to status here, and include discharge and ward as optional arguments?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -178,11 +193,12 @@ _Details coming soon ..._
...
-| **Find**   | `find name/NAME` or `find nric/NRIC`<br> e.g., `find name/John`                                   |
-| **List**   | `list`                                                                                            |
-| **Help**   | `help`                                                                                            |
+| Action       | Format, Examples                                                                           |
+| ------------ | ------------------------------------------------------------------------------------------ |
+| **Add**      | `add nric/NRIC name/NAME [s/STATUS]​` <br> e.g., `add nric/S1234567A name/John Doe s/GREEN |
+| **Delete**   | `delete INDEX`<br> e.g., `delete 1`                                                        |
+| **Edit**     | `edit INDEX [s/STATUS] [w/WARD] [d/DISCHARGE]​`<br> e.g.,`edit 1 s/GREEN`                  |
+| **Find**     | `find name/NAME` or `find nric/NRIC` or `find s/STATUS`<br> e.g., `find name/John`         |
+| **Add Ward** | `addward name/NAME [c/CAPACITY` <br> e.g., `addward name/S1234567A c/25`                   |
```
Comment (nramapurath) [inline docs/UserGuide.md:202]: Is this missing a closing square bracket after CAPACITY?

File: docs/UserGuide.md (hunk 3 above)
Comment (jeraldkiew) [reply docs/UserGuide.md:78]: changed to status, discharge and ward can only edit

Comment (nramapurath) [reply docs/UserGuide.md]: Oh right. My bad!

**Pull request merged**